### ISSUE TITLE
It looks `await-thenable` is not existing anymore

### ISSUE THREAD

**Issue by Ky6uk** [typescript-eslint/typescript-eslint#436]
`README` in this repository has the rule `await-thenable`,  but ESLint throws an error when you try to use it.

**Comment by bradzacher** [comment]
Hi @Ky6uk - could you please give us some more information?
We have an issue template for a reason.

Please fill out the below template:

```
<!--
If you have a problem with a specific rule, please begin your issue title with [rulename] to make it easier to search for.
I.e. "[no-unused-vars] False positive when fooing the bar"
-->

**Repro**

<!--
Include a minimal reproduction case.
Please try to avoid code that isn't directly related to the bug, as it makes it harder to investigate.
-->

```JSON
{
  "rules": {
    "typescript/<rule>": "<setting>"
  }
}
```

```TS
// your repro code case
```

**Expected Result**

**Actual Result**

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `X.Y.Z` |
| `@typescript-eslint/parser`        | `X.Y.Z` |
| `TypeScript`                       | `X.Y.Z` |
| `ESLint`                           | `X.Y.Z` |
| `node`                             | `X.Y.Z` |
| `npm`                              | `X.Y.Z` |
```

**Comment by Ky6uk** [comment]
Sorry, I thought it pretty clear. I just tried to use rule `@typescript-eslint/await-thenable` from `README` file of `@typescript-eslint/typescript-eslint` repository and got an error.

**Comment by merlinnot** [comment]
This rule was merged to master, but there was no release yet, so it's not available in the latest npm package.

**Comment by bradzacher** [comment]
@Ky6uk - the problem is that your definition of "pretty clear" is different from ours.

Questions that you didn't answer in OP, but are in the template, and would have helped me answer your question instantly:
- what is your eslint version?
  - if you're using eslint v4 then this will cause the rule to be not found.
- what is your `@typescript-eslint/eslint-plugin` version?
  - if you're using the master version 1.6.0 then the rule hasn't been published there yet.
- what is your config?
  - if you've mistyped the rule name, or the plugin name, then this could cause the rule to not be found.

Please don't ignore the template because you think it's clear - repos implement them for a reason - to help collect as much information up-front as possible and save the back-and-forth delay of going back to the OP for more information.
Chances are (as with this issue) there's information you don't realise is relevant that you're omitting by ignoring the template.

Not trying to be rude, sorry - it's just this added several comments of noise when this could have been closed with one response.

---

With that aside, @merlinnot is correct - the rule has been merged into master, but not yet released.

We release to `main` every week or so - this is a manual process so it gets done when someone gets around to it.
If you want to be on the bleeding edge, please use the `canary` tag on npm, which is automatically built and released for every single master commit.
